### DISCUSSION TITLE
Cypress crash

### DISCUSSION THREAD

**Discussion by QAbdel** [cypress-io/cypress#28809] [Questions and Help]
Plzz need big help! all my tests run correctly and the tool works, but one of my tests the tool apparently has a bug;  and only for the one with a payment, as soon as it finishes it returns to the home page of the tool (cypress_bibl, where there are the specs) without error without any indication and the video will be more explicit (I have  simply stop at the penultimate step and finish it manually on the tool and in both cases (manu or auto) I have the same behavior

Fyi: when i click on "non" , its continue correctly and continue and load pages After .
 PI: I reinstalled the tool to fix it but without success.

**Comment by MikeMcC399** [comment]
This seems like a duplicate question to https://discord.com/channels/755913899261296641/1201130307382939798
